src/sonic_ax_impl/mibs/vendor/cisco/ciscoEntityFruControlMIB.py

```diff
@@ -0,0 +1,99 @@
+import imp
+import re
+import sys
+
+from sonic_ax_impl import mibs
+from ax_interface import MIBMeta, ValueType, MIBUpdater, MIBEntry, SubtreeMIBEntry
+from ax_interface.encodings import ObjectIdentifier
+
+PSU_PLUGIN_MODULE_NAME = 'psuutil'
+PSU_PLUGIN_MODULE_PATH = "/usr/share/sonic/platform/plugins/{}.py".format(PSU_PLUGIN_MODULE_NAME)
+PSU_PLUGIN_CLASS_NAME = 'PsuUtil'
+
+# TODO: Remove once PSU util will be available directly
+sys.path.append("/usr/lib/python2.7/dist-packages/")
+
```

Comment:
This repo uses python3.6 only. Please don't mix.

Reply:
Yes, that is reason for such #TODO because PSU plugin uses python2.7 as all utilities. 
It's required to import PSUBase module.

Reply:
Don't mix. This is not acceptable.

Reply:
@qiluo-msft: Do you have a suggestion as to how to proceed? I don't think it's reasonable to build sonic-utilities into both Python 2 and Python 3 packages for this, which is just a temporary solution.

Reply:
How about the permanent solution?

Reply:
After this https://github.com/Azure/sonic-buildimage/pull/1212, you could directly import it